calculator/Calculator.java

```diff
@@ -0,0 +1,56 @@
+package calculator;
+
+import java.util.*;
+public class Calculator {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		Scanner sc=new Scanner(System.in);
+		System.out.println("Enter first number:");
+		double a=sc.nextDouble();
+		System.out.println("Enter second number:");
+		double b=sc.nextDouble();
+		System.out.println("Give any one operation(+,-,*,/)");
+		char c=sc.next().charAt(0);
+		double result;
+		try {
+			switch(c) {
+				case '+':
+				{
+					result=a+b;
+					System.out.println("Addition of two numbers: "+ result );
+					break;
+				}
+				case '-':
+				{
+					result=a-b;
+					System.out.println("Subtraction of two numbers: "+result);

```

Comment:
Move all the operations to a separate method.
Refer : https://www.w3schools.com/java/java_methods.asp